decentdao/decent-interface
# Roles 0.2.0/flush out the stream on wearer change

This PR introduces following changes/functionality:

- Fetch and Display Payroll and Vesting information for the given Hat Account
- Rinse out payroll/vesting stream to the current wearer when changing hat wearer
- Rinse out payroll/vesting stream to the current wearer and cancel stream, which automatically returns remaining asset balance to the Safe, when deleting hat

Testing:
1. If you were creating payroll before - make sure it appears in the Roles list.
2. Try changing the wearer on the hat with active payroll (aka start date and first tranche already passed). Expected result - funds that weren't claimed are transferred to the "current wearer" and then wearer is changed to the new wearer
3. Try deleting the hat with active payroll. Expected result - funds that weren't claimed are transferred to the "current wearer", remaining funds from the stream are transferred back to the Safe, stream is cancelled, and then hat is "deleted"
